BUILD.bazel

```diff
@@ -91,6 +91,15 @@ filegroup(
     }),
 )
 
+filegroup(
+    name = "android_symbols",
+    srcs = [
+        "//gapidapk/android/apk:arm64-v8a_gapir.sym",
+        "//gapidapk/android/apk:armeabi-v7a_gapir.sym",
+        "//gapidapk/android/apk:x86_gapir.sym",
+    ],
+)
+
```

Comment:
Change to:
```
filegroup(
  name = "symbols",
  srcs = [
    "//cmd/gapir/cc:gapir.sym",
  ] + select({
   "//tools/build:linux": [
       "//gapidapk/android/apk:arm64-v8a_gapir.sym",
       "//gapidapk/android/apk:armeabi-v7a_gapir.sym",
       "//gapidapk/android/apk:x86_gapir.sym",
    ],
    "//conditions:default": [],
  }),
)
```

Then change the build scripts on all platforms to build the symbols target, so that we can control what gets built from the build files, rather than mucking around with the scripts.